arch/arm/src/armv8-r/arm_doirq.c

```diff
@@ -58,7 +58,9 @@ uint32_t *arm_doirq(int irq, uint32_t *regs)
    * and we will use this_task(). Therefore, it cannot be overridden.
    */
 
+#ifdef CONFIG_SMP
```

Comment:
split to another PR

Reply:
no, this is regression of arm_doirq()

Reply:
Why not, this not the introduce by https://github.com/apache/nuttx/pull/13651/files#diff-723f44e76d5ebc519459ee93d6d0e42a7cea30ad0343b61e3ab48a491f2aaf58